src/ApiService/ApiService/onefuzzlib/ConfigOperations.cs

```diff
@@ -26,7 +26,7 @@ private sealed record InstanceConfigCacheKey();
     private static readonly InstanceConfigCacheKey _key = new(); // singleton key
     public Task<InstanceConfig> Fetch()
         => _cache.GetOrCreateAsync(_key, async entry => {
-            entry = entry.SetAbsoluteExpiration(TimeSpan.FromMinutes(10)); // cached for 10 minutes
+            entry = entry.SetAbsoluteExpiration(TimeSpan.FromMinutes(1)); // cached for 1 minute
```

Comment:
10 minutes was wrong ? or you forgot to change back after testing ?

Reply:
No I'm changing it permanently. With the instance_config, we want the changes to be immediate. It doesn't make sense to update the instance_config and then wait for ten minutes to see the changes.

Reply:
By the comments I understand this to be the cached value. So we don't wait to see the changes, but we're deciding how long the cache is good for before invalidating it and going back to storage to fetch again.